<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminGenres</class>
 <widget class="QMainWindow" name="AdminGenres">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Жанры</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="leGenre">
       <property name="placeholderText">
        <string>Введите жанр</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QPushButton" name="btnDel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="btnSave">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbGenre">
       <property name="text">
        <string>Жанр</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnMenu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Меню</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>411</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlTable">
     <item>
      <layout class="QHBoxLayout" name="hlSearch">
       <item>
        <widget class="QLineEdit" name="leSearch">
         <property name="placeholderText">
          <string>Введите жанр</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSearch">
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="twData">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnAddMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить жанр</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
